cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -3179,6 +3179,18 @@ def _migrate_attached_cross_vc(self, context, dest_host, volume, backing):
                     CREATE_PARAM_DISK_LESS: True
                 })
             return (True, None)
+        except exceptions.DuplicateName:
+            # The backing could have already been created by a
+            # previous (failed) migration attempt.
+            disk_devices = self.volumeops._get_disk_devices(backing)
```

Comment:
Why do we have a backing already that we can check, when we call `create_backing()` above to create one? I would think we would have to query the target vCenter for the already-existing shadow-VM and probably cannot use whatever `backing` points to.